pkg/restapi/vc/operation/models.go

```diff
@@ -61,6 +62,22 @@ type IssueCredentialOptions struct {
 	AssertionMethod string `json:"assertionMethod,omitempty"`
 }
 
+// ComposeCredentialRequest for composing and issuing credential.
+type ComposeCredentialRequest struct {
+	Issuer                  string                 `json:"issuer,omitempty"`
+	Subject                 string                 `json:"subject,omitempty"`
+	Types                   []string               `json:"types,omitempty"`
+	IssuanceDate            *time.Time             `json:"issuanceDate,omitempty"`
+	ExpirationDate          *time.Time             `json:"expirationDate,omitempty"`
+	Claims                  map[string]interface{} `json:"claims,omitempty"`
+	Evidence                verifiable.Evidence    `json:"evidence,omitempty"`
```

Comment:
nitpick: I believe part of the point of a "compose" API is to decouple the credential format from the inputs. Therefore, `Evidence` should not be typed as `verifiable.Evidence`.

Reply:
Missed this. Makes sense, will add it in next PR.